source/v4/system.yaml

```diff
@@ -14,3 +14,22 @@
             $ref: "#/definitions/StatusOK"
         '400':
           $ref: '#/responses/BadRequest'
+
+  '/config':
+    put:
+      tags:
+        - system
+      summary: Update the Server configuration
```

Comment:
Can the summary be `Update configuration` and description `Submit a new configuration for the server to use.`